powerauth-push-server/src/main/java/io/getlime/push/service/PushMessageSenderService.java

```diff
@@ -208,7 +208,7 @@ public void onSuccess(ResponseEntity<String> stringResponseEntity) {
         });
     }
 
-    // If application exists return list of devices related to that app
+    // If user exists return list of devices related to that app
```

Comment:
I do not understand purpose of this method - comment is inconsistent with method signature, method returns APNS client.